Gemfile

```diff
@@ -17,8 +17,6 @@ gem 'turbolinks',                 '5.2.1'
 gem 'jbuilder',                   '2.10.0'
 gem 'bootsnap',                   '1.4.6', require: false
 gem 'devise'
-gem 'sendgrid-ruby'
-gem 'figaro'
```

Comment:
Gems were originally considered for use with email verification, but weren't used in the end